src/mocks/entity/mockDataset.ts

```diff
@@ -85,13 +106,15 @@ const versionInfo: VersionInfo[] = [
 
 const bundle: EntityBundle = {
   entity: mockDatasetEntity,
+  entityType: EntityType.DATASET,
```

Comment:
Bundle would be successfully fetched in EntityModal but entityType would be undefined